platform/ios/src/MGLSDKUpdateChecker.mm

```diff
@@ -0,0 +1,37 @@
+#import "MGLSDKUpdateChecker.h"
+#import "NSBundle+MGLAdditions.h"
+
+@implementation MGLSDKUpdateChecker
+
++ (void)checkForUpdates {
+#if TARGET_IPHONE_SIMULATOR
+    // Abort if running in a playground.
+    if ([[NSBundle mainBundle].bundleIdentifier hasPrefix:@"com.apple.dt.playground."]) {
```

Comment:
Did not see evidence of the update check being called in IB, but IB is also failing to load our designable anyway... so I added the process exclusion in 470c6c0.